docs/k8s-quickstart.asciidoc

```diff
@@ -263,15 +262,11 @@ spec:
 EOF
 ----
 
-[id="{p}-deep-dive"]
-== Deep dive
-
-Now that you have completed the quickstart, you can try out more features like using persistent storage.
-
 [float]
+[id="{p}-persistent-storage"]
 === Use persistent storage
 
-The cluster that you deployed in this quickstart uses an link:https://kubernetes.io/docs/concepts/storage/volumes/#emptydir[emptyDir volume], which might not qualify for production workloads.
+Now that you have completed the quickstart, you can try out more features like using persistent storage. The cluster that you deployed in this quickstart uses a default persistent volume claim of 1GiB, without a storage class set. This means that the default storage class defined in the Kubernetes cluster is the one that will be provisioned.
```

Comment:
I think that's not true for 0.8, we should probably fallback to the emptyDir mention.

Reply:
Reverted back to the previous text.